arangopipe/tests/.spyproject/workspace.ini

```diff
@@ -0,0 +1,10 @@
+[workspace]
+restore_data_on_startup = True
```

Comment:
what is .spyproject? do we need this checked in?

Reply:
removed it, we dont need it